### PR TITLE
Make "genesis" blob origin explicit.

### PR DESCRIPTION
**Pull request by afck**
## Motivation

There is another spurious "blobs not found" error: When we request a timeout certificate on a _root_ chain that does not have any blocks yet. That's because root chain descriptions are the only blobs that don't have a blob _state_.

## Proposal

Add a blob state for them, and make the "genesis" origin explicit.

## Test Plan

A regression test was added.

## Release Plan

- Would be nice to fix this on `testnet_conway` somehow, too. On the other hand, it's a very rare edge case.

## Links

- [reviewer checklist](https://github.com/linera-io/linera-protocol/blob/main/CONTRIBUTING.md#reviewer-checklist)

### CODE REVIEW

File: linera-storage/src/lib.rs [hunk 1]
```diff
@@ -263,9 +263,21 @@ pub trait Storage: linera_base::util::traits::AutoTraits + Sized {
...
+        let description_blob = Blob::new_chain_description(&description);
+        let description_blob_id = description_blob.id();
+        self.write_blob(&description_blob).await?;
+        self.maybe_write_blob_states(
+            &[description_blob_id],
+            BlobState {
+                origin: BlobOrigin::Genesis,
+                last_used_by: None,
+                epoch: None,
+            },
```
Comment (deuszx) [inline linera-storage/src/lib.rs]: Or simpler, `BlobState::genesis()` as the helper. Unless we don't need it anywhere else. Actually, it could be some kind of `const`.

Comment (afck) [reply linera-storage/src/lib.rs]: Done.

**Review by deuszx — APPROVED** [review]
